content/learn/book/storing-data/required-components.md

```diff
@@ -6,6 +6,59 @@ weight = 2
 status = 'hidden'
 +++
 
-- You, the user, are not required to put these on. They are added implicitly by the engine.
-- You can specify them on your components using the macro
-- You can find the required components by looking at the component impl
+In the previous section, we saw how simple components can be composed to form complex
+entities. Often times, a given component will need certain other components in order to do anything
+useful. Drawing from the example in the previous section, it probably doesn't make much sense to
+mark an entity as a `Combatant` if it doesn't also have `Life`.
+
+To help prevent errors of omission, and to simplify the task of spawning, Bevy lets you declare
+that a given component depends on the presence of another component:
+
+```rust,hide_lines=1-5
+# use bevy::ecs::prelude::*;
+
+#[derive(Component)]
+struct Life(u8);
+
+#[derive(Component)]
+#[require(Life)]
+struct Combatant;
+```
+
+Here, we've told the engine that `Combatant` requires `Life`: "Whenever you insert a `Combatant`
+component, insert a `Life` component as well."
+
+However, we didn't specify in the `require` clause how _much_ `Life` we should give the new entity.
```

Comment:
I think it's worth explicitly mentioning that if you don't specify how much `Life` then you Bevy inserts `Life::default()`.

Reply:
Good idea; I'm currently working on another chapter so I'll get to it later.

Reply:
Done